src/__stories__/circle-story.tsx

```diff
@@ -20,6 +20,12 @@ export const Default: StoryComponent = () => {
             <div data-testid="circle-with-image">
                 <Circle size={40} backgroundImage={avatarImg}></Circle>
             </div>
+            <div data-testid="circle-with-border-default">
+                <Circle size={40} backgroundColor={skinVars.colors.brand} border />
```

Comment:
can you change `backgroundColor={skinVars.colors.brand}` to `backgroundColor={skinVars.colors.background}`

Reply:
done